### DISCUSSION TITLE
Grouping in Track List

### DISCUSSION THREAD

**Discussion by pro-fy** [quodlibet/quodlibet#4512] [Ideas]
Hi all,

I would really like to see the feature to group the track list, e. g. by album. Track lists with lots of tracks would become clearer :-)

In DeaDBeeF and Foobar this is possible.

![image](https://github.com/quodlibet/quodlibet/assets/95591643/c47da9bc-177e-4a36-8591-cef6dc5226ad)

A setting that allows for grouping would be great. Screenshot shows the settings dialog from DeaDBeef.

![image](https://github.com/quodlibet/quodlibet/assets/95591643/24980021-29b0-4f28-81e9-b840fae2a804)

What do you think?

Cheers

**Comment by declension** [comment]
Yes, this is something that's come up a few times. Would be good but a fair bit of work given how deeply integrated the songlist is. See #3315.
